### PR TITLE
Add cscope utility

### PR DESCRIPTION
**Pull request by cgrzemba**
cscope-fast is containted in onbld tools but the general tool cscope is usefull for browsing C projects in general.
So please add this utility.

### CODE REVIEW

File: build/cscope/build.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+. ../../lib/functions.sh
+
+PROG=cscope
+VER=15.9
+PKG=ooce/developer/cscope
+SUMMARY="Cscope is a developer's tool for browsing source code"
+DESC="Cscope is a developer's tool for browsing source code. It has an impeccable"
+DESC+="Unix pedigree, having been originally developed at Bell Labs back in the"
+DESC+="for many years, and has been used to manage projects involving 20 million"
```
Comment (oetiker) [inline build/cscope/build.sh:26]:
```suggestion
DESC+="Unix pedigree, having been originally developed at Bell Labs, "
DESC+="and has been used to manage projects involving 20 million " 
```
or find the missing bit of text :)

Comment (cgrzemba) [reply build/cscope/build.sh]: found the missing line :)

**Comment by hadfl** [comment]
hi @cgrzemba, I put a few small changes on top of your commit: https://github.com/hadfl/omnios-extra/commit/8d4dd419fde8af125ec18d611d2339a2fdbe9bb9

Please have a look and let us know if it is ok for you if we quash them with your commit and get it integrated.

**Comment by hadfl** [comment]
closed in favour of https://github.com/omniosorg/omnios-extra/pull/942

**Pull request closed without merging**